src/main.rs

```diff
@@ -107,6 +118,57 @@ fn main() {
 
 }
 
+/// Traverse current directory and search for dominant
+/// language using tokei.
+fn get_dominant_language() -> Option<Language> {
+    let mut languages = tokei::Languages::new();
+    let required_languages = get_all_language_types();
+    languages.get_statistics( &["."], vec![".git", "target"], Some(required_languages));
+
+    languages.remove_empty().iter()
+                            .max_by_key(|(_, v)| v.code)
+                            .map(|(k, _)| Language::from(**k))
+}
+
+/// Convert from tokei LanguageType to known Language type .
+impl From<tokei::LanguageType> for Language {
+    fn from(language: tokei::LanguageType) -> Self {
+        match language {
+            tokei::LanguageType::Rust => Language::Rust,
+            tokei::LanguageType::Go => Language::Go,
+            tokei::LanguageType::Java => Language::Java,
+            tokei::LanguageType::Cpp => Language::Cpp,
+            tokei::LanguageType::C => Language::C,
+            tokei::LanguageType::Python => Language::Python,
+            tokei::LanguageType::CSharp => Language::Csharp,
+            tokei::LanguageType::Scala => Language::Scala,
+            tokei::LanguageType::Sh => Language::Shell,
+            tokei::LanguageType::Lisp => Language::Lisp,
+            tokei::LanguageType::Haskell => Language::Haskell,
+            tokei::LanguageType::Ruby => Language::Ruby,
+            tokei::LanguageType::R => Language::R,
+            _ => unimplemented!(),
```

Comment:
JavaScript is a valid `LanguageType`.

cf https://docs.rs/tokei/6.0.2/tokei/enum.LanguageType.html

Reply:
Of course, this is a work in progress. An issue was opened #2 for that purpose.

Feel free to contribute @Mouvedia 

Thanks 🥇